File: js/fib.js
// terribly inefficient implementation of Fibonacci Sequence
function fib(n) {
    if (n == 0 || n == 1) {
        return 1;
    }
    else {
        return fib(n-1) + fib(n-2);
    }
}

var a = parseInt(process.argv[2]);
console.log( fib(a) );

File: java/fib.java
class Fib {

    public static int fib(int n) {
        if (n == 0 || n == 1) {
            return 1;
        }
        else {
            return Fib.fib(n-1) + Fib.fib(n-2);
        }
    }

	public static void main(String[] args) {
        String a = args[0];
        int i = Integer.parseInt(a);
        System.out.println(Fib.fib(i));
	}
}

